### PR TITLE
fix: bin/nodetool eval rpc case clause

### PR DESCRIPTION
**Pull request by JimMoen**
Fixes <issue-or-jira-number>

```bash
$ emqx eval 'emqx:get_config([logs]).'
escript: exception error: no case clause matching {error,{config_not_found,[logs]}}
  in function  nodetool__escript__1732__246864__885259__2:do/1 {FILE_PATH}
  in call from escript:start/1 (escript.erl, line 277)
  in call from init:start_em/1 
  in call from init:do_boot/3 
```

Release version: v/e5.?

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: bin/nodetool [hunk 1]
```diff
@@ -176,6 +176,9 @@ do(Args) ->
             case rpc:call(TargetNode, emqx_ctl, run_command, [eval_erl, Parsed], infinity) of
                 {ok, Value} ->
                     io:format("~p~n", [Value]);
+                {error, Reason} ->
+                    io:format("RPC to ~s error: ~p\n", [TargetNode, Reason]),
+                    halt(1);
                 {badrpc, Reason} ->
                     io:format("RPC to ~p failed: ~p~n", [TargetNode, Reason]),
```
Comment (zmstone) [inline bin/nodetool:183]:
```suggestion
                    io:format("RPC to ~s failed: ~p~n", [TargetNode, Reason]),
```

File: bin/nodetool [hunk 2]
```diff
@@ -176,6 +176,9 @@ do(Args) ->
             case rpc:call(TargetNode, emqx_ctl, run_command, [eval_erl, Parsed], infinity) of
                 {ok, Value} ->
                     io:format("~p~n", [Value]);
+                {error, Reason} ->
+                    io:format("RPC to ~s error: ~p\n", [TargetNode, Reason]),
```
Comment (zmstone) [inline bin/nodetool:180]:
```suggestion
                    io:format("RPC to ~s error: ~p~n", [TargetNode, Reason]),
```

**Review by zmstone — COMMENTED** [review]
minor comment for code style consistency

**Review by zmstone — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-6678)

**Pull request merged**